modules/httpcache/httpcache.go

```diff
@@ -17,16 +17,21 @@ import (
 )
 
 // AddCacheControlToHeader adds suitable cache-control headers to response
-func AddCacheControlToHeader(h http.Header, d time.Duration) {
+func AddCacheControlToHeader(h http.Header, d time.Duration, additionalDirectives []string) {
+	directives := make([]string, 0, 2+len(additionalDirectives))
+
 	if setting.IsProd {
-		h.Set("Cache-Control", "private, max-age="+strconv.Itoa(int(d.Seconds())))
+		directives = append(directives, "private")
+		directives = append(directives, "max-age="+strconv.Itoa(int(d.Seconds())))
```

Comment:
what does a `max-age=0` do?

Reply:
Disable caching, thought it's basically the same outcome as`no-store`. We certainly never want to have HTML content cached.

Reply:
Huh, that seems like a wrong quote, but we could interpret it as `no-store`.

Reply:
added it